### ISSUE TITLE
Midpoint Review - Accessibility Feedback - IIR Team, Veteran Status Card, New Home & User Experience

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#102799]
## Next Steps for the VFS team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/gb1dfnkl) at any time._

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- Sorry for the very delayed feedback --- and thank you to @DavidPearlUX for the gentle nudge in Slack! Not sure whether this got lost because of GitHub or because of user error. In any case, thank you very much for your patience, and please don't hesitate to reach out if you have any questions or concerns.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [x] **Must:** When displaying the "Add to wallet" elements, make sure those are displayed based on detected device at all screen sizes, not just the mobile viewports. There are a number of accessibility situations where that's relevant, but my go-to here is that some users will use a tablet as their primary device because the screen is easier to see or the device size is easier for them to handle. Best to not make assumptions about what "mobile device" means and just detect whether iOS or Android are in use.
- [x] **Should:** Rather than directly printing the paper copy of the status card, I'd recommend using the download variant of `va-link` to point to a PDF file, which can then be saved or printed at the discretion of the Veteran. The link text should be clear about the interaction --- "download a printable PDF" or similar, rather than "print."
   - [ ] **Alternative must:** If you decide that you'd rather directly print the status card by triggering the browser's print function (no PDF download), then use a `va-button` rather than an link. Since it's not navigating you to a file and is instead triggering an action, the button is the right element here.
- [ ] **Should:** @artsymartha68 brought up the many different variations on dark mode that should be accounted for. I don't currently have a rigorous standard for what kinds of dark modes/contrast modes should be tested against, and VA.gov doesn't currently style for dark mode directly the way the mobile app does. But if you're looking for a deep dive into the world of various "dark modes" and CSS, here are several different "modes" and the CSS media flags that go with them. They each behave a little bit differently, which is super fun and not at all frustrating. I'd recommend testing in as many as you can and styling anything that seems problematic, with the understanding that new standards for how we design for each of these may come later.
   - Windows: **Dark Mode** (styled by [prefers-color-scheme](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme)), **High Contrast Mode** (styled by [forced-colors](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/forced-colors)).
   - Mac: **Dark Mode** (styled by [prefers-color-scheme](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme)).
   - iOS: **Dark Mode** (styled by [prefers-color-scheme](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme)), **Increase Contrast** (styled by [prefers-contrast](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-contrast)), **Smart Invert** (styled by [inverted-colors](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/inverted-colors)), and **Classic Invert** (does not seem to be a way to style it!).
   - Android: **Dark Mode** (styled by [prefers-color-scheme](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme)), **Color Inversion** (styled by [inverted-colors](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/inverted-colors)).

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone

**Comment by DavidPearlUX** [comment]
Thank you @briandeconinck ! I have made the relevant [adjustments to the figma file](https://www.figma.com/design/AobGKkBvIiDuYXR2QqkytC/IIR-Veteran-Status?node-id=3197-33151&t=dyq5H2F6NXpR9Bem-1) and will continue to work with our Engineers to ensure proper color contrast and reading of device.
